### ISSUE TITLE
Valhingen: Need a sling of seeking

### ISSUE THREAD

**Issue by grannypron** [grannypron/uaf_levels#13]
Valhingen: Need a sling of seeking for GiveTreasure #33 @ 15,8 - replace the sling+3 with it

**Comment by grannypron** [comment]
This walkthrough describes it as:

https://gamefaqs.gamespot.com/pc/564785-pool-of-radiance/faqs/73869

"Sling of Seeking +2 -- this grants its bonus both to hit and to damage (though
  this is not actually special in Pool of Radiance, because magical bows
  incorrectly do the same)."

**Comment by grannypron** [comment]
Tough to decide if I should just make a copy of Sling +2 with a different name or just use a Sling +2.  I think it might be cool to have names that are the same as the PoR game, even if it is a bit misleading.  Still adds some fun to the game I think.  

In the end, I decided to just leave it as a Sling +2 to stay consistent with what I did with the Long Sword +2 Flame Tongue (issue #5), which I think should not be faked, because it would be even more misleading to the player.

Commit is f865ffe
